### ISSUE TITLE
winsound.MessageBeep doesn't play on core failure

### ISSUE THREAD

**Issue by LeonarddeR** [nvaccess/nvda#18193]
### Steps to reproduce:
1. From source, in louis/__init__.py or another file, create a syntax error
2. Start NVDA from source

### Actual behavior:
NVDA logs a critical core failure, but doesn't play the critical error sound.

### Expected behavior:
NVDA logs a critical core failure and plays the critical error sound.

### Additional context
Swapping the `winsound.MessageBeep` call back to `winsound.PlaySound("SystemHand", winsound.SND_ALIAS | winsound.SND_ASYNC)` plays the sound again. This indicates that this is a regression caused by #7543

**Comment by tspivey** [comment]
I can't reproduce that here. Are your system sounds working?

**Comment by tspivey** [comment]
If it's the same issue I'm thinking of, there's a Windows bug which causes system sounds to sometimes not work on reboot. The only workaround I've found, besides rebooting and hoping they come back, is to run this command elevated:

`schtasks /run /tn \Microsoft\Windows\Multimedia\SystemSoundsService`

I don't know what triggers this bug, but it also causes `MessageBeep` not to work. Running that command while the system sounds are working will turn them off.

**Comment by LeonarddeR** [comment]
The problem is not with MessageBeep. When I call log.critical from the python console, it just beeps fine. It's only in this critical core failure case that it doesn't want to play.

**Comment by SaschaCowley** [comment]
I also can't repro this

**Comment by LeonarddeR** [comment]
Interesting, I can repro this on two different systems, at least from the master branch. Haven't tested this from beta.

**Comment by SaschaCowley** [comment]
@LeonarddeR can you please provide your Windows and Python versions?

**Comment by LeonarddeR** [comment]
I'm at Python 3.11.9 x86. Windows 11 24h2 build 26100.4061.

**Comment by LeonarddeR** [comment]
I found the underlying issue. I'm using a different sound scheme that also uses sounds for close program and open program. Whenever the critical stop sound is played, it is directly followed by a close program sound. It is probably common windows behavior that playing another message beep suppresses a playing beep. This wasn't the case in the old situation.
